bindings/c/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ endif()
 
 if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "arm64")
   set(cpu "aarch64")
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
```

Comment:
Could you replace this check with:
`elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "^(ppc64le|powerpc64le)")`

While not tested yet, this should allow to build on FreeBSD.

Reply:
Thank you!